pyproject.toml

```
@@ -177,9 +178,9 @@ run-mypy = { cmd = "mypy virtualizarr" }
 # is used), reduces test hangs that appear to be macOS-related.
 run-tests = { cmd = "pytest -n auto --dist loadscope --run-network-tests --verbose" }
 run-tests-no-network = { cmd = "pytest -n auto --verbose" }
-run-tests-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov=term-missing" }
-run-tests-xml-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov-report=xml" }
-run-tests-html-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov-report=html" }
+run-tests-cov = { cmd = "pytest -n 1 --run-network-tests --verbose --cov=virtualizarr --cov=term-missing" }
+run-tests-xml-cov = { cmd = "pytest -n 1 --run-network-tests --verbose --cov=virtualizarr --cov-report=xml" }
+run-tests-html-cov = { cmd = "pytest -n 1 --run-network-tests --verbose --cov=virtualizarr --cov-report=html" }
```

```suggestion
run-tests-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov=term-missing" }
run-tests-xml-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov-report=xml" }
run-tests-html-cov = { cmd = "pytest -n auto --run-network-tests --verbose --cov-report=html" }
```
Not sure if you meant to include these changes?